ci/testgrid/Makefile

```diff
@@ -13,8 +13,17 @@
 # limitations under the License.
 
 TESTGRID_DIR := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
+GENERATOR_DIR := ../prow
+
+config:
+	go run ${GENERATOR_DIR}/make_config.go --generate-prow-config=false --testgrid-config-output="config.yaml" ${GENERATOR_DIR}/config_knative.yaml
```

Comment:
Better call `make config` on ci/prow. It reduces duplicate code and ensure both configs are in sync.